src/sentry/replays/lib/storage.py

```diff
@@ -34,6 +38,50 @@ class RecordingSegmentStorageMeta:
     file: File | None = None
 
 
+def make_recording_filename(segment: RecordingSegmentStorageMeta) -> str:
+    return _make_recording_filename(
+        segment.retention_days,
+        segment.project_id,
+        segment.replay_id,
+        segment.segment_id,
+    )
+
+
+def make_video_filename(segment: RecordingSegmentStorageMeta) -> str:
+    return _make_video_filename(
+        segment.retention_days,
+        segment.project_id,
+        segment.replay_id,
+        segment.segment_id,
+    )
+
+
+def _make_recording_filename(
+    retention_days: int | None,
+    project_id: int,
+    replay_id: str,
+    segment_id: int,
+) -> str:
+    """Return a recording segment filename."""
+    return "{}/{}/{}/{}".format(
+        retention_days or 30,
+        project_id,
+        replay_id,
+        segment_id,
+    )
+
+
+def _make_video_filename(
+    retention_days: int | None,
+    project_id: int,
+    replay_id: str,
+    segment_id: int,
+) -> str:
+    """Return a recording segment video filename."""
+    filename = _make_recording_filename(retention_days, project_id, replay_id, segment_id)
+    return filename + ".video"
```

Comment:
should it be .video over .mp4? i assume so, but want to make sure

Reply:
I think so.  We don't actually know what kind of file it is (in this endpoint).  The client knows the file type because the RRWeb recording data contains it.  If we statically assume mp4 we'll be naming it with the correct extension but if the video format changes in the future (or on different SDKs) then the extension becomes meaningless.

I used the ".video" extension which might be a mistake.  Maybe it should be in a video "sub-folder" instead.

Reply:
i think this makes sense - extension isn't super important here i think